src/app/main-page/main-page.component.scss

```diff
@@ -0,0 +1,62 @@
+// Define color variables
+$primaryColor: #528be7;
```

Comment:
only variable that are common , and we are using them a lot in the project we will do variable .not all the value. pleae check if all variables are relevant

Reply:
fixed